buildtest/builders/base.py

```diff
@@ -930,6 +930,23 @@ def _get_variables(self, variables):
 
         return lines
 
+    def _extract_content(self, regex):
+        """Extract content based on the stream and linenum properties
+        from the regex field and return it as a string.
+
+        Args:
+            regex (dict): regex section from the metrics field
+        """
+
+        stream = regex.get("stream")
+        content = self._output if stream == "stdout" else self._error
+
+        linenum = regex.get("linenum")
+        if linenum is not None and content:
+            split = content.split("\n")
```

Comment:
this is confusing please dont use variable `split` when doing `content.split("\n")`

instead we should use a more meaningful variable such as `line`

Reply:
👍